default.nix

```diff
@@ -426,9 +423,9 @@ rec {
       '';
     };
 
-    unit = ocamlTestDerivation {
+    unit = testDerivation {
```

Comment:
it doesn't seem necessary to run unit tests in `musl` (static) link environment. @nomeata do you agree? It still passes and won't pull in duplicate OCaml library versions into the `moc-shell`.